File: data/code_pairs/cpp/572.cpp
#include <iostream>
#include <string>

// Function prototype
void testStringLength(const std::string& testString, size_t expectedLength);

int main() {
    std::string testString = "Hello";
    size_t expectedLength = 5;

    testStringLength(testString, expectedLength);
    return 0;
}

void testStringLength(const std::string& testString, size_t expectedLength) {
    size_t actualLength = testString.length();
    if (actualLength == expectedLength) {
        std::cout << "C++ Test PASSED." << std::endl;
    } else {
        std::cout << "C++ Test FAILED." << std::endl;
    }
}

File: data/code_pairs/fortran/572.f90
program main
  implicit none
  character(len=10) :: testString = "Hello"
  integer :: expected_length, actual_length

  expected_length = 5
  actual_length = len_trim(testString)

  call testLength(actual_length, expected_length)
end program main

subroutine testLength(actual_length, expected_length)
  implicit none
  integer, intent(in) :: actual_length, expected_length

  if (actual_length == expected_length) then
     print *, "Fortran Test PASSED."
  else
     print *, "Fortran Test FAILED."
  end if
end subroutine testLength